sbt/src/sbt-test/project/sbt-version-change/build.sbt

```diff
@@ -0,0 +1,6 @@
+TaskKey[Unit]("checkSbtVersionWarning") := {
+	val state = Keys.state.value.globalLogging
+	val contents = IO.read(state.backing.file)
+	assert(contents.contains("SBT version in build.properties file has changed, use 'reboot' instead of 'reload' to use the new value"))
```

Comment:
Change here too?

Reply:
Oops, forgot about that one, sorry! Fixed.